### ISSUE TITLE
"Apps are 64KB. They don't fit in the available -1024KB." error

### ISSUE THREAD

**Issue by metawops** [numworks/epsilon-sample-app-cpp#27]
When I try to install the `voord.nwa` file (from the release here) I get the above error message in Chrome:

<img width="967" alt="Screenshot 2022-08-27 at 23 11 07" src="https://user-images.githubusercontent.com/376090/187048098-2b398afc-9ba8-4398-8db3-08c3526b50f0.png">

What can I do? Why the error? Why the (obviously wrong) reporting of -1024 KB?

**Comment by Ecco** [comment]
Is your calculator up to date?

**Comment by metawops** [comment]
Yes, updated it prior to this test to the latest version (19.4.1).
As I was an early adopter I have a N0100 hardware revision, unfortunately. Aren't those compatible?

**Comment by Ecco** [comment]
That's the problem indeed. Unfortunately they're not as the OS takes pretty much every single bit of available Flash memory!

We definitely should add a warning though. Thanks for your help!

**Comment by metawops** [comment]
Oh! So I'll have to spend another 100 Euros to get all the latest features and be able to add additional apps? That's too bad. Makes me a bit angry, tbh. My early invest in your startup (then) brings me to a point where I have to invest again. Weird. 🤷‍♂️ (Won't do it, of course.)
But thanks for the clarifications.

**Comment by Ecco** [comment]
If anything, given how many [software updates](https://www.numworks.com/calculator/update/) we have released over time, the device you own today is vastly superior to the one you bought several years ago, even though you weren't charged a single additional cent for those improvements.

We did roll out new hardware revisions in the meantime, and those may be eligible for additional features. You absolutely don't have to buy one of those.
